shared/actions/team-building.tsx

```diff
@@ -181,18 +181,31 @@ const fetchUserRecs = async (
     const contactRes = _contactRes || []
     const contacts = contactRes.map(
       (x): TeamBuildingTypes.User => ({
+        assertion: x.assertion,
         contact: true,
         id: x.assertion,
+        keybaseUsername: x.username,
         label: x.displayLabel,
         prettyName: x.displayName,
-        serviceMap: {keybase: x.username},
+        serviceMap: {},
+        serviceName: 'contact',
```

Comment:
I think this is used for icons in user box and maybe some other things as well

Reply:
but `contact: true` might not be needed anymore then

Reply:
This RPC should also be changed to return RPCTypes.UserSearchResult (or similar) instead of RPCTypes.ProcessedContact. It would also include `bubbleText` (note: it will be renamed to something else though in general)